apps/workflowengine/lib/Check/FileMimeType.php

```diff
@@ -68,6 +68,29 @@ public function setFileInfo(IStorage $storage, $path) {
 		}
 	}
 
+	/**
+	 * The mimetype is only cached if the file exists. Otherwise files access
+	 * control will cache "application/octet-stream" for all the target node on:
+	 * rename, move, copy and all other methods which create a new item
+	 *
+	 * To check this:
+	 * 1. Add an automated tagging rule which tags on mimetype NOT "httpd/unix-directory"
+	 * 2. Add an access control rule which checks for any mimetype
+	 * 3. Create a folder and rename it, the folder should not be tagged, but it is
+	 *
+	 * @param string $storageId
+	 * @param string|null $path
+	 * @param string $mimeType
+	 * @return string
+	 */
+	protected function cacheAndReturnMimeType(string $storageId, ?string $path, string $mimeType): string {
```

Comment:
```suggestion
	protected function cacheAndReturnMimeType(string $storageId, string $path, string $mimeType): string {
```

@rullzer @juliushaertl @nickvergessen this is not working on server15!
This pr have been merged with failing jobs: https://drone.nextcloud.com/nextcloud/server/22464/5/3

Reply:
yeah, just remove `?string` and I think we need to remove `string` aswell?